src/commands/addons/config.js

```diff
@@ -39,7 +39,7 @@ class AddonsConfigCommand extends Command {
       return false
     }
 
-    // TODO update getAddonManifest to https://open-api.netlify.com/#/default/showServiceManifest
+    // TODO update getAddonManifest to https://open-api.netlify.com/#/operation/showServiceManifest
```

Comment:
It looks like these links don't work with the `/` after the `#`
```suggestion
    // TODO update getAddonManifest to https://open-api.netlify.com/#operation/showServiceManifest
```